E3SM-2-0/check/proof

```diff
@@ -0,0 +1,120 @@
+../1pctCO2_r1i1p1f1.json:  "experiment_id": "1pctCO2",
+../abrupt-4xCO2_r1i1p1f1.json:  "experiment_id": "abrupt-4xCO2",
+../abrupt-4xCO2_r2i1p1f1.json:  "experiment_id": "abrupt-4xCO2",
+../amip_r1i1p1f1.json:  "experiment_id": "amip",
+../amip_r2i1p1f1.json:  "experiment_id": "amip",
+../amip_r3i1p1f1.json:  "experiment_id": "amip",
+../hist-aer_r1i1p1f1.json:  "experiment_id": "hist-aer",
+../hist-aer_r2i1p1f1.json:  "experiment_id": "hist-aer",
+../hist-aer_r3i1p1f1.json:  "experiment_id": "hist-aer",
+../hist-aer_r4i1p1f1.json:  "experiment_id": "hist-aer",
+../hist-aer_r5i1p1f1.json:  "experiment_id": "hist-aer",
+../hist-GHG_r1i1p1f1.json:  "experiment_id": "hist-GHG",
+../hist-GHG_r2i1p1f1.json:  "experiment_id": "hist-GHG",
+../hist-GHG_r3i1p1f1.json:  "experiment_id": "hist-GHG",
+../hist-GHG_r4i1p1f1.json:  "experiment_id": "hist-GHG",
+../hist-GHG_r5i1p1f1.json:  "experiment_id": "hist-GHG",
+../historical_r1i1p1f1.json:  "experiment_id": "historical",
+../historical_r2i1p1f1.json:  "experiment_id": "historical",
+../historical_r3i1p1f1.json:  "experiment_id": "historical",
+../historical_r4i1p1f1.json:  "experiment_id": "historical",
+../historical_r5i1p1f1.json:  "experiment_id": "historical",
+../piClim-control_r1i1p1f1.json:  "experiment_id": "piClim-control",
+../piClim-histaer_r1i1p1f1.json:  "experiment_id": "piClim-histaer",
+../piClim-histaer_r2i1p1f1.json:  "experiment_id": "piClim-histaer",
+../piClim-histaer_r3i1p1f1.json:  "experiment_id": "piClim-histaer",
+../piClim-histall_r1i1p1f1.json:  "experiment_id": "piClim-histall",
+../piClim-histall_r2i1p1f1.json:  "experiment_id": "piClim-histall",
+../piClim-histall_r3i1p1f1.json:  "experiment_id": "piClim-histall",
+../piControl_r1i1p1f1.json:  "experiment_id": "piControl",
+ 
+../1pctCO2_r1i1p1f1.json:  "history": "v2.LR.historical_0101",
```

Comment:
should fix history

Reply:
This one DOES need fixing, according to my checker.